client/src/styles/_variables.scss

```diff
@@ -0,0 +1,24 @@
+// Fonts
+$font-primary: 'Montserrat', 'Arial', sans-serif;
+
+// rwd breakpoints
+$breakpoint-xs-max: 575px;
+$breakpoint-sm-max: 767px;
+$breakpoint-md-max: 991px;
+$breakpoint-xl-max: 1366px;
+$breakpoint-xxl-max: 1600px;
+
+// Colors
+$color-primary: blue;
+$color-secondary: yellow;
+
+// Text Color
+$text-color-primary: #000;
+$text-color-secondary: aqua;
+
+// Fonts sizes
+$font-size-primary: 32px;
```

Comment:
I'd say it's way too big to be primary. Maybe `large`? 🤔

Reply:
I just created the file. Those values are not final - everyone can enter the appropriate value whenever it is determined :)